cmd/lotus-miner/sealing.go

```diff
@@ -410,7 +410,7 @@ var sealingDataCidCmd = &cli.Command{
 		},
 	},
 	Action: func(cctx *cli.Context) error {
-		if cctx.NArg() < 1 || cctx.NArg() > 2 {
+		if cctx.NArg() != 1 || cctx.NArg() > 2 {
```

Comment:
no, this check won't work, `!=1||>2` is only going to catch `1` but it can be 2, leave this one alone